content/javascript/RegEx.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "RegEx"
+weight = 1
```

Comment:
Weights determine the order of pages in the sidebar. Having two pages with the same weight sorts them alphabetically IIRC. Is this what you intended? Also adding an `updated` field here would be nice for both of these files. Ex: `updated = 2022-03-26`

Reply:
Oh cool, I did not realize.  Good to know.